packages/schema/src/Schema.ts

```diff
@@ -5746,6 +5747,209 @@ export class DateFromNumber extends transform(
   { strict: true, decode: (n) => new Date(n), encode: (d) => d.getTime() }
 ).annotations({ identifier: "DateFromNumber" }) {}
 
+/**
+ * Describes a schema that represents a `DateTime.Utc` instance.
+ *
+ * @category DateTime.Utc constructors
+ * @since 0.68.26
+ */
+export class DateTimeUtcFromSelf extends declare(
+  (u) => dateTime.isDateTime(u) && dateTime.isUtc(u),
+  {
+    identifier: "DateTimeUtcFromSelf",
+    description: "a DateTime.Utc instance",
+    pretty: (): pretty_.Pretty<dateTime.Utc> => (dateTime) => dateTime.toString(),
+    arbitrary: (): LazyArbitrary<dateTime.Utc> => (fc) => fc.date().map((date) => dateTime.unsafeFromDate(date)),
+    equivalence: () => dateTime.Equivalence
+  }
+) {}
+
+const decodeDateTime = <A extends dateTime.DateTime.Input>(input: A, _: ParseOptions, ast: AST.AST) =>
+  ParseResult.try({
+    try: () => dateTime.unsafeMake(input),
+    catch: () => new ParseResult.Type(ast, input)
+  })
+
+/**
+ * Defines a schema that attempts to convert a `number` to a `DateTime.Utc` instance using the `DateTime.unsafeMake` constructor.
+ *
+ * @category DateTime.Utc transformations
+ * @since 0.68.26
+ */
+export class DateTimeUtcFromNumber extends transformOrFail(
+  Number$,
+  DateTimeUtcFromSelf,
+  {
+    strict: true,
+    decode: decodeDateTime,
+    encode: (dt) => ParseResult.succeed(dateTime.toEpochMillis(dt))
+  }
+).annotations({ identifier: "DateTimeUtcFromNumber" }) {}
+
+/**
+ * Defines a schema that attempts to convert a `string` to a `DateTime.Utc` instance using the `DateTime.unsafeMake` constructor.
+ *
+ * @category DateTime.Utc transformations
+ * @since 0.68.26
+ */
+export class DateTimeUtc extends transformOrFail(
+  String$,
+  DateTimeUtcFromSelf,
+  {
+    strict: true,
+    decode: decodeDateTime,
+    encode: (dt) => ParseResult.succeed(dateTime.formatIso(dt))
+  }
+).annotations({ identifier: "DateTime.Utc" }) {}
```

Comment:
As an `identifier`, it's better, if possible, to use a value that is a valid TypeScript identifier (in this case "DateTimeUtc" rather than "DateTime.Utc"). This is because, for example, it might be used by a compiler to produce a TypeScript type or to generate code.